File: Python/Human.py
class Human():
    # parent class
    
    # atribut privat
    __nik = ""
    __name = ""
    __gender = ""
    
    # constructor
    def __init__(self, nik="", name="", gender=""):
      self.__nik = nik
      self.__name = name
      self.__gender = gender
    
    #getter setter
    def setNik(self, nik):
      self.__nik = nik
    
    def getNik(self):
      return self.__nik
    
    def setName(self, name):
      self.__name = name 
    
    def getName(self):
      return self.__name
    
    def setGender(self, gender):
      self.__gender = gender   
    
    def getGender(self):
      return self.__gender

File: CPP/Program/Human.cpp
#include "header.hh"

// Constructor
Human::Human()
{
    nik = "";
    name = "";
    gender = "";
}

// Contructor dengan atribut
Human::Human(string nik, string name, string gender)
{
    this->nik = nik;
    this->gender = gender;
    this->name = name;
}

// Getter Setter
string Human::getNIK()
{
    return nik;
}

void Human::setNIK(string nik)
{
    this->nik = nik;
}

string Human::getName()
{
    return name;
}

void Human::setName(string name)
{
    this->name = name;
}

string Human::getGender()
{
    return gender;
}

void Human::setGender(string gender)
{
    this->gender = gender;
}

// Destructor
Human::~Human()
{
}